subsys/bluetooth/host/l2cap.c

```diff
@@ -1049,8 +1049,8 @@ static void le_conn_param_rsp(struct bt_l2cap *l2cap, struct net_buf *buf)
 {
 	struct bt_l2cap_conn_param_rsp *rsp = (void *)buf->data;
 
-	if (buf->len < sizeof(*rsp)) {
-		LOG_ERR("Too small LE conn param rsp");
+	if (buf->len != sizeof(*rsp)) {
+		LOG_ERR("Invalid LE conn param rsp size");
```

Comment:
```suggestion
		LOG_ERR("Invalid LE conn param rsp size (%u != %zu)", buf->len, sizeof(*rsp));
```

Generally it's nice to the values in the log as well for debugging purposes